From 94438e044ca58fdfecb4429cb64f5df949ac7d98 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:27:10 -0400
Subject: [alsa] hda: Make azx_attach_pcm_stream static

Message-id: <140972823573-221-patchset-jkysela@redhat.com>
Patchwork-id: 90921
O-Subject: [RHEL 7.1 PATCH 220/442] ALSA: hda - Make azx_attach_pcm_stream static
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

It is only used in hda_controller.c now.

Signed-off-by: Dylan Reid <dgreid@chromium.org>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 7c3e438add2710f9a92017e2d942081db4f88909)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/hda_controller.c b/sound/pci/hda/hda_controller.c
index 6156d0a..97993e1 100644
--- a/sound/pci/hda/hda_controller.c
+++ b/sound/pci/hda/hda_controller.c
@@ -971,8 +971,8 @@ static void azx_pcm_free(struct snd_pcm *pcm)
 
 #define MAX_PREALLOC_SIZE (32 * 1024 * 1024)
 
-int azx_attach_pcm_stream(struct hda_bus *bus, struct hda_codec *codec,
-     struct hda_pcm *cpcm)
+static int azx_attach_pcm_stream(struct hda_bus *bus, struct hda_codec *codec,
+     struct hda_pcm *cpcm)
 {
  struct azx *chip = bus->private_data;
  struct snd_pcm *pcm;
-- 
1.7.1